src/private/meetingRoom.ts

```diff
@@ -0,0 +1,245 @@
+import { ensureInitialized, sendMessageRequestToParent } from '../internal/internalAPIs';
+import { GlobalVars } from '../internal/globalVars';
+import { SdkError } from '../public/interfaces';
+
+export namespace meetingRoom {
+  /**
+   * @private
+   * Hide from docs
+   *
+   * Data structure to represent a meeting room.
+   */
+  export interface MeetingRoomInfo {
+    /**
+     * Endpoint id of the meeting room.
+     */
+    endpointId: string;
+    /**
+     * Device name of the meeting room.
+     */
+    deviceName: string;
+    /**
+     * Client type of the meeting room.
+     */
+    clientType: string;
+    /**
+     * Client version of the meeting room.
+     */
+    clientVersion: string;
+  }
+
+  /**
+   * @private
+   * Hide from docs
+   *
+   * Enum used to indicate meeting room capabilities.
+   */
+  export enum Capability {
+    /**
+     * Media control capability: toggle mute.
+     */
+    TOGGLE_MUTE = 'toggleMute',
```

Comment:
Why are we using all-caps names for these enum values? This is not standard for JavaScript and doesn't match the rest of the SDK. Please stick with upper camel case notation (i.e. ToggleMute, ToggleCamera, etc.).

Reply:
Changed to upper camel case.